AquaME.lua

```diff
@@ -268,8 +253,8 @@ simplePlain = AquaME {
 
     model.channelDepth = 2.5    -- Additional depth at the highest border of the channel, in meters.
     model.rainBegin = 0         -- Instant when when rain begins after the beginning of the simulatio, in hours.
-    model.rainDuration = 5     -- Rain duration, in hours.
-    model.rainRate = 0.005      -- Rate of rain, in mm per hour.
+    model.rainDuration = 10     -- Rain duration, in hours.
+    model.rainRate = 0.015      -- Rate of rain, in m per hour.
```

Comment:
is it "m" or "mm"?

Reply:
It is m (meters). This comment was wrong in the previous versions.